server/db.ts

```diff
@@ -95,6 +95,19 @@ export class Database {
         this.save();
     }
 
+    remove_user_from_group(group_id: number, user_id: number) {
+        let my_group = this.groups[group_id].members;
+        let i:any;
+        for (i in my_group) {
```

Comment:
careful using for..in, use for..of instead.

for..in loops over the fields of the object, this would include i="length", which is probably not intentional